compiler/one-cmds/tests/one-infer_007.test

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -d option is invalid, -b is accepted
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+inputfile="sample.tvn"
+
+if [[ ! -s "${inputfile}" ]]; then
+  touch ${inputfile}
+fi
+
+# copy dummy-infer to bin folder
+cp dummy-infer ../bin/dummy-infer
+
+invalid_driver="invalid-infer"
+
+# run test
+one-infer -d ${invalid_driver} -b dummy -- ${inputfile} > ${filename}.log
```

Comment:
this is tricky. as you wrote, if the `-d` is invalid we can search for `-b`. or as `-d` is invalid we can raise error and stop, or show error message and continue.
for the user, although `-d` is invalid and the tool continues, user may not notice there is an error. and this may confuse users. (or some users may think the tool is clever)
for me, I would go with raise error and stop.